src/System.Private.CoreLib/shared/System/Numerics/Vector4_Intrinsics.cs

```diff
@@ -133,9 +157,15 @@ public Vector4(Vector3 value, float w)
         /// </summary>
         /// <param name="other">The Vector4 to compare this instance to.</param>
         /// <returns>True if the other Vector4 is equal to this instance; False otherwise.</returns>
+#if ARM64
         [Intrinsic]
+#endif
         public readonly bool Equals(Vector4 other)
         {
+            if (Sse.IsSupported)
+            {
+                return Sse.MoveMask(Sse.CompareEqual(this.AsVector128(), other.AsVector128())) == 0b1111;
```

Comment:
`Sse.MoveMask(Sse.CompareNotEqual(v1, v2)) == 0;` ? (`test` instead of `cmp`). I think you told me this trick a while ago 🙂

Reply:
Fixed. 😄